source/_integrations/squeezebox.markdown

```diff
@@ -61,6 +61,68 @@ Volume step:
  description: Amount to adjust the volume when turning volume up or down.  
 {% endconfiguration_basic %}
 
+## Announce
+
+The Squeezebox media player entity supports the "announce" parameter. When media is played with announce:true, the current state of the media player is saved, the media is then played, and when playing is finished, the original state is restored. For example, if the media player is on and playing a track, once the announcement is finished, the track will resume playing at the same point it was paused by the announcement. If the media player was off, it will be turned off again after playing the announcement.
+
+### Extra Keys
+
+The following extra keys are available to modify the announcement
+
+| Data attribute     | Optional | Description                                                                                               |
+| ------------------ | -------- | --------------------------------------------------------------------------------------------------------- |
+| `announce_volume`  | yes      | Specify the volume of the announcement                                                                    |
+| `announce_timeout` | yes      | Specify the maximum length of the announcement in seconds after which the original media will be resumed. |
```

Comment:
Similarly, let's specify the value range and what happens if this parameter isn't passed in.